### DISCUSSION TITLE
Lazy loading of releases/repositories/environment

### DISCUSSION THREAD

**Discussion by jehutyy** [helmfile/helmfile#693] [Questions]
Is there a way to load specific part of a helmfile.yaml upon execution ? 

I would like to add specific repository without having to load every others.

For instance, I have a helm repository for which I have to set environment variable even though I'm not even using a release that needs it.

Let's say I have 100 repositories (and 100 releases), I would like to only load repositories used by release I'm installing (with -l selector).

**Comment by yxxhero** [comment]
Maybe this a feature request. For performance. @mumoshu @xiaomudk @itscaro

**Comment by xiaomudk** [comment]
Let's discuss how to implement it specifically.
